lagunita/cms/static/sass/partials/custom_overrides/_header.scss

```diff
@@ -0,0 +1,6 @@
+@media screen and (min-width: 1280px) {
+    #su-header .su-container,
+    #su-footer .su-container {
+        width: 1280px !important;
```

Comment:
I'm using `!important` here intentionally. This CSS gets applied before our included stanford one...

Reply:
Per below, we should just move this into `su-identity.css`.

Besides the benefits mentioned there,
we'd no longer need the `!important` because we can just add the override where it makes sense;
we can just place that after the existing `1200px` rule in `su-identity.css` :)

This also makes things more maintainable, as you'd find all the styling for the header/footer
in one place, the existing file.
It seems weird to split this up, if we can avoid it.

All we need is the proper prefix for CMS, which it looks like we can get like:

```css
@media screen and (min-width: 1280px) {
  .wrapper #su-header .su-container,
  .wrapper #su-footer .su-container {
    width: 1280px;
  }
}
```